configure

```diff
@@ -632,8 +649,24 @@ cat > $rootdir/config.erl << EOF
 EOF
 
 
+function install_local_rebar(){
+  if [ ! -x "${rootdir}/bin/rebar" ]; then
+      git clone --branch 2.6.0 https://git-wip-us.apache.org/repos/asf/couchdb-rebar.git ${rootdir}/src/rebar
```

Comment:
hm, I think that's just git being weird.

try `git clone --depth 1 --branch 2.6.0 https://git-wip-us.apache.org/repos/asf/couchdb-rebar.git`, it's faster, we don't need the history.